### DISCUSSION TITLE
is this project alive or dead?

### DISCUSSION THREAD

**Discussion by devinschumacher** [QQuick/Transcrypt#842] [Q&A]
what is the status of transcrypt? is it still being worked on or...?

**Comment by JennaSys** [comment]
Yes definitely. My plans to move it forward got stalled a bit these last few months, but are still going to happen.

**Reply by devinschumacher** [reply]
okay awesome bc i love the concept.

are you looking for help? im currently growing a community of AI/ML programmers (which is obv. mostly python & js people). theres bound to be someone in there that would be interested in contributing. you're welcome to join the discord if you'd like. --- [Discord link here](https://devin.to/discord)

**Reply by JennaSys** [reply]
Eventually I'll likely need some help or at least guidance by someone that has more in-depth JS knowledge than I do. For now, it's on me to get the ball rolling.

**Reply by justinbarak** [reply]
@JennaSys Hey, I'm looking to get started with transcrypt myself. Is it worth holding off now until you drop the next release? Do I need to stick with python 3.9 or can I update to a newer version while I wait for the release?

**Reply by JennaSys** [reply]
Use Python 3.9 for now. I plan on pushing some updates/fixes soon, but updating the Python version to match the AST is a bigger task and probably won't happen right away. FWIW, I'll be targeting 3.11 or 3.12 depending on when I can get to it.
